### ISSUE TITLE
training loss increases and keeps near 3.26+07 in my own dataset

### ISSUE THREAD

**Issue by jzhang538** [TiantianWang/ICCV17_SRM#2]
I clone the git, install the PSP net version caffe, and also download the pretrained model. After I change the path to my own dataset, I use the bash code:
$CAFFE_ROOT/build/tools/caffe train --solver=$SOLVER_ROOT/solver.prototxt --gpu=2 --weights=$MODEL_ROOT/snapshot_iter_0_twostage_init.caffemodel

to train a model. However, the loss at iteration 0 is 644313, iteration 1 is 2.49185e+07 and then it keeps near 3.26389e+07.  Also, I find that the learning rate is 1e-10. When I change it to 1e-1, the loss becomes nan. Whether there's any problem with my training process?

Note: for the predicting process with your trained model, everything works successfully.
